index.js

```diff
@@ -3,6 +3,124 @@ const nav = document.querySelector('.nav-wrapper');
 const closeBtn = document.querySelector('.close-menu');
 const ancreLinks = nav.getElementsByTagName('a');
 
+const theModal = document.querySelector('#myModal');
+const closeModal = document.querySelector('.closeModal');
+const worksSection = document.querySelector('.work-container');
+
+const data = [
+  {
+    name: 'Tonic',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi architecto 
+    mollitia, earum fugiat possimus expedita! Qui nostrum blanditiis est, cupiditate autem 
+    fugiat non consequatur consectetur itaque dolor, quo deserunt sequi.
+    Consequuntur placeat, delectus eius nemo veritatis eaque quas. Nulla minima, harum nobis 
+    odio, in non placeat voluptas provident vitae, quaerat maxime minus amet. Libero nisi 
+    maiores molestias repudiandae natus provident.`,
+    featuredImage: './image/snapshoot.png',
+    technologies: ['html', 'css', 'javaScript'],
+    linkToLive: '',
+    linkToSource: '',
+    cardNo: '#first-card',
+  },
+  {
```

Comment:
- [ ] nice work please in the desktop version, kindly align the see source button with the image like in the figma version.

yours
![Screenshot (687)_LI](https://user-images.githubusercontent.com/65298048/208944074-c47dceea-dc67-483b-950c-9d4e5137ff0a.jpg)
figma
![Screenshot (688)](https://user-images.githubusercontent.com/65298048/208944176-baa0085b-5551-43a4-9af1-2a50a93aa9cb.png)